### ISSUE TITLE
Gap between hierarchical node's boundary and edge's start point (2)

### ISSUE THREAD

**Issue by uruuru** [eclipse/elk#686]
Follow-up of #546, as that one fixed the issue only partly. 

![image](https://user-images.githubusercontent.com/7552459/89677046-facf6a00-d8ec-11ea-9921-51c84f29b112.png)

```
{
  id: "root",
  layoutOptions: { 'algorithm': 'layered' },
  children: [
    {
      id: "n1",
      width: 30,
      height: 30,
      layoutOptions: { "portConstraints": "FIXED_SIDE", "elk.direction": "DOWN" },
      ports: [
        {
          id: "p1",
          width: 15,
          height: 15,
          layoutOptions: { "port.side": "NORTH" }
        },
        {
          id: "p2",
          width: 15,
          height: 15,
          layoutOptions: { "port.side": "SOUTH" }
        }
      ],
      children: [
        {
          id: "n2",
          width: 30,
          height: 30,
          ports: [
            {
              id: "p3",
              width: 15,
              height: 15,
              layoutOptions: { "port.side": "WEST" }
            },
            {
              id: "p4",
              width: 15,a
              height: 15,
              layoutOptions: { "port.side": "EAST" }
            }
          ],
        },
      ],
      edges: [
        { id: "e1", sources: [ "p3" ], targets: [ "p1" ] },
        { id: "e2", sources: [ "p2" ], targets: [ "p4" ] }
      ]
    },
  ],
}
```

**Comment by le-cds** [comment]
I cannot seem to reproduce the problem. This is the layout I'm getting:

![image](https://user-images.githubusercontent.com/7465584/99152405-ce707a00-26a1-11eb-8a36-ef78dd318f99.png)

Obtained with the following ELKT, which should specify (almost) identical input:

```
algorithm: layered

node compound {
	algorithm: layered
	elk.direction: DOWN
	portConstraints: FIXED_SIDE
	
	port pNorth {
		layout[ size: 15, 15 ]
		^port.side: NORTH
	}
	port pSouth {
		layout[ size: 15, 15 ]
		^port.side: SOUTH
	}
	
	node child {
		layout[ size: 30, 30 ]
		
		port p1 {
			layout[ size: 15, 15 ]
		}
		port p2 {
			layout[ size: 15, 15 ]
		}
	}
	
	edge child.p1 -> compound.pNorth
	edge compound.pSouth -> child.p2
}
```

**Comment by uruuru** [comment]
I locally verified that with a elkjs build based on the current master, the issue is indeed fixed.

**Comment by uruuru** [comment]
I'll open for a moment to remind me to verify this on the 0.7.1 branch.

**Comment by uruuru** [comment]
Looks fine on the release branch.

**Comment by uruuru** [comment]
Likely fixed by either #682 or #680.
